src/DDoSX/RecordClient.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Please could we add the shortcut method onto `UKFast\SDK\DDoSX\Client`?

Reply:
Added method